queries/move-to-LoketLB-graphs.sparql

```diff
@@ -0,0 +1,216 @@
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX org: <http://www.w3.org/ns/org#>
+PREFIX lblodlg: <http://data.lblod.info/vocabularies/leidinggevenden/>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX extlmb: <http://mu.semte.ch/vocabularies/ext/lmb/>
+PREFIX schema: <http://schema.org/>
+PREFIX person: <http://www.w3.org/ns/person#>
+PREFIX temp: <http://mu.semte.ch/graph/temp/>
+
+# Inserting bestuurseenheid
+# Should stay in public graph
+# INSERT {
+#   GRAPH ?targetGraph {
+#     ?bestuurseenheid ?p ?o.
+#     ?os ?op ?bestuurseenheid.
+#   }
+# } WHERE {
+#   GRAPH <http://mu.semte.ch/graphs/public> {
+#     ?bestuurseenheid a besluit:Bestuurseenheid;
+#                      mu:uuid ?id.
+#     {
+#       # Outgoing
+#       ?bestuurseenheid ?p ?o.
+#       VALUES ?p { rdf:type mu:uuid skos:prefLabel skos:altLabel ext:wilMailOntvangen ext:mailAdresVoorNotificaties ext:isTrailUser ext:viewOnlyModules
+#                   besluit:werkingsgebied ext:inProvincie besluit:classificatie }
+#     }
+#     UNION
+#     {
+#       # Incoming
+#       ?os ?op ?bestuurseenheid.
+#       VALUES ?op { ext:contactVoor besluit:bestuurt ext:origineleBestuurseenheid }
+#     }
+#   }
+#   BIND(IRI(CONCAT("http://mu.semte.ch/graphs/organizations/", ?id, "/LoketLB-mandaatGebruiker")) AS ?targetGraph)
+# };
+# Inserting bestuurseenheid-contact
+# bestuurseenheid-contact doesn't exit in the loket data
+
+# Inserting werkingsgebied
+# Should stay in public graph
+# INSERT {
+#   GRAPH ?targetGraph {
+#     ?werkingsgebied ?p ?o.
+#   }
+# } WHERE {
+#   GRAPH ?targetGraph {
+#     ?bestuurseenheid besluit:werkingsgebied ?werkingsgebied.
+#   }
+#   GRAPH <http://mu.semte.ch/graphs/public> {
+#     ?werkingsgebied a prov:Location;
+#                     ?p ?o.
+#     VALUES ?p { rdf:type mu:uuid rdfs:label ext:werkingsgebiedNiveau }
+#   }
+# };
+# UNDER CONSTRUCTION, IN THE PROCESS OF ADDING ext:inGraph
+# Inserting bestuursorgaan
+INSERT {
+  GRAPH ?targetGraph {
+    ?orgaan ?p ?o.
+    ?fakeOrgaan ?pp ?oo.
+    ?os ?op ?orgaan.
+    ?oss ?opp ?fakeOrgaan.
+    ?orgaan ext:inGraph ?targetGraph.
```

Comment:
it's odd that you both add all predicates to the targetgraph AND keep track of in which graph the subject should be. The second part seems very redundant given that you already moved the triples.

Since you are transforming a loket db here, you will have a lot of graphs that need to be dropped. As you can't drop them by query, you will need to write some sort of script to drop them based on a query. e.g.

1. select all irrelevant graphs
2. loop over these graphs and
2.1. drop silent graph the bad graph

Reply:
Keeping track of the target graph is something I do for the types coming after bestuursorgaan, like (should be type) bestuursorgaan in tijd, mandaat, RechtstreekseVerkiezing etc.

The script I will look into